driver/generic/st77xx.py

```diff
@@ -457,7 +458,7 @@ def __init__(self,doublebuffer=True,factor=4):
         self.disp_drv = lv.disp_create(self.width, self.height)
         self.disp_drv.set_flush_cb(self.disp_drv_flush_cb)
         self.disp_drv.set_draw_buffers(bytearray(bufSize), bytearray(bufSize) if doublebuffer else None, bufSize, lv.DISP_RENDER_MODE.PARTIAL)
-        self.disp_drv.set_color_format(lv.COLOR_FORMAT.NATIVE if self.bgr else lv.COLOR_FORMAT.NATIVE_REVERSED)
+        self.disp_drv.set_color_format(lv.COLOR_FORMAT.NATIVE)
```

Comment:
do not use `lv.COLOR_FORMAT.NATIVE` that is what the `color_format` parameter is for. That is what should be passed when setting the color format

Reply:
Fixed.
Because this is ST77xx, the color format is explicitly set to RGB565.